simple-backend/nlpviewer_backend/models.py

```diff
@@ -1,11 +1,36 @@
 from django.db import models
 
+class Project(models.Model):
+    name = models.CharField(max_length=200)
+    ontology = models.TextField(default='')
+
+    # relationship: document -- defined in Document
```

Comment:
Coding style: commented discarded code can be removed during PR.

Reply:
Comments are rewritten.